pkgs/applications/networking/irc/catgirl/default.nix

```
@@ -0,0 +1,21 @@
+{lib, stdenv, fetchurl, ncurses, libressl, man, pkg-config, ctags}:
+
+stdenv.mkDerivation rec {
+  name = "catgirl-1.6";
+
+  src = fetchurl {
+    url = "https://git.causal.agency/catgirl/snapshot/${name}.tar.gz";
+    sha256 = "0shg02zidqqmvywqqsaazlgg9rd5lhhrvjx6n0lzmdfaawxywciv";
+  };
+
+  nativeBuildInputs = [ pkg-config ncurses libressl man ctags ];
+
+  meta = with lib; {
+    homepage = "https://git.causal.agency/catgirl/about/";
+    license = lib.licenses.gpl3;
+    description = "A TLS-only terminal IRC client";
+    platforms = lib.platforms.unix;
```

See line 13 (`with lib`)

```suggestion
    platforms = platforms.unix;
```